mixer/adapter/lightstep/client.go

```diff
@@ -0,0 +1,74 @@
+package lightstep
+
+import (
+	"fmt"
+	"istio.io/api/mixer/adapter/model/v1beta1"
+	"istio.io/istio/mixer/template/tracespan"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+
+	"github.com/lightstep/lightstep-tracer-go/collectorpb"
+)
+
+var _ collectorpb.CollectorServiceClient = &grpcSatelliteClient{}
+
+// grpcSatelliteClient specifies how to send reports back to a LightStep
+// Satellite via grpc.
+type grpcSatelliteClient struct {
+	options    ClientOptions
+	grpcClient collectorpb.CollectorServiceClient
+}
+
+type ClientOptions struct {
+	// ReporterID the ID of a particular Adapter client
+	ReporterID uint64
```

Comment:
This shouldn't be an option I think. If I look at the lightstep tracer, it is a random number generated when the tracer is initialized.